presto-main/src/test/java/com/facebook/presto/operator/TestMergeHashSort.java

```diff
@@ -56,12 +58,17 @@ public void testChannelIterator()
         assertFalse(iterator.hasNext());
     }
 
+    private LocalMemoryContext aMemoryContext()
```

Comment:
I like aMemoryContext better is clearly says that it's just "an" object not some particular one, and whether I create it or return the same one it doesn't matter to me at this point